schema/sale.js

```diff
@@ -10,6 +12,18 @@ import {
   GraphQLBoolean,
 } from 'graphql';
 
+export function auctionState({ start_at, end_at }) {
```

Comment:
I noticed this because there's some random auction that's not open in staging saying it is:

```
      {
        "id": "juliens-auctions-street-art-now",
        "auction_state": "open",
        "start_at": "02/12/16",
        "end_at": "02/22/16"
      }
```

https://github.com/artsy/force/blob/master/models/sale.coffee#L52